pkg/rke/k8s_rke_system_images.go

```diff
@@ -9932,6 +10016,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.8.2
+		"v1.27.8-rancher2-3": {
```

Comment:
The same comment as for `v1.26.11-rancher2-3`

Reply:
As discussed offline, an additional version is needed to provide an option to upgrade to ACI 6.0.3.3 vs. doing (what is a feature upgrade) to 6.0.4.1.